eng/pipelines/templates/stages/archetype-cpp-release.yml

```diff
@@ -100,6 +100,24 @@ stages:
                           Write-Host "##vso[task.setvariable variable=PrBranchName]$branchName"
                         displayName: Set fork branch name
 
+                      - pwsh: |
+                          $packageSpec = Get-Content -Raw -Path "$(Pipeline.Workspace)/packages/${{artifact.Name}}/package-info.json" | ConvertFrom-Json
+                          $version = $packageSpec.version
+                          $title = "[${{ artifact.VcpkgPortName }}] publish version $version"
+
+                          if ('$(VcpkgPRTitle)') { 
+                            Write-Host "Using queue time PR title"
+                            $title = '$(VcpkgPRTitle)'
```

Comment:
Why do we see this block execute in the logs, when we don't set `VcpkgPRTitle` during a run:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=728450&view=logs&j=099dde79-2162-5ef6-b009-e03f78c45d18&t=1de584cf-18c9-5dfb-9d46-6241ccd4dd35
```text
Using queue time PR title
Finishing: Set PR title
```

Also, at what point do we set this `$title` and pass it to the `create pull request` script as its `PRTitle` argument?